app/assets/stylesheets/ama_layout/layout_components/progress-bar.scss

```diff
@@ -0,0 +1,137 @@
+.progress-bar{
+  @include hide-for(small only);
+  overflow: auto;
+
+  &__section{
+    &--active{
+      background: $cerulean;
+    }
+
+    &--inactive{
+      background: $brand-blue-light;
+    }
+
+    &--completed{
+      background: $brand-blue-dark;
+    }
+  }
+
+  &--selected{
+    background: $cerulean;
+    color: $white;
+  }
+
+  &--completed{
+    color: $white;
+    background: $brand-blue-dark;
+  }
+
+  &__section-quarter{
+    @include medium-3;
+    position: relative;
+    color: $white;
+    overflow: hidden;
+    padding: 6px 0;
+  }
+    &__section-title{
+
+    }
+
+    &__section-info{
+      display: block;
+    }
+
+  &__arrow{
+    width: 32px;
+    height: 32px;
+    position: absolute;
+    right: -20px;
+    top: -4px;
+
+    @include breakpoint(medium down){
+      display: none;
+    }
+
+    &:after{
+      content: '';
+      position: absolute;
+      right: 0;
+      border-color: transparent transparent transparent $brand-blue-light;
+      border-style: solid;
+      border-width: 35px;
+    }
+
+    &:before{
+      content: '';
+      position: absolute;
+      right: -4px; //this controls the thickness of the arrow
+      border-color: transparent transparent transparent $white;
+      background: $brand-blue-light;
+      border-style: solid;
+      border-width: 35px;
+    }
+
+    &--selected:after{
+      content: '';
+      color: $white;
+      position: absolute;
+      right: 0;
+      border-color: transparent transparent transparent $cerulean;
+      border-style: solid;
+      border-width: 35px;
+    }
+
+    &--selected:before{
+      content: '';
+      color: $white;
+      position: absolute;
+      right: -4px; //this controls the thickness of the arrow
+      border-color: transparent transparent transparent $white;
+      background: $brand-blue-light;
+      border-style: solid;
+      border-width: 35px;
+    }
+
+    &--completed:after{
+      content: '';
+      color: $white;
+      position: absolute;
+      right: 0;
+      border-color: transparent transparent transparent $brand-blue-dark;
+      border-style: solid;
+      border-width: 35px;
+    }
+
+    &--completed:before{
+      content: '';
+      color: $white;
+      position: absolute;
+      right: -4px; //this controls the thickness of the arrow
+      border-color: transparent transparent transparent $white;
+      background: $cerulean;
+      border-style: solid;
+      border-width: 35px;
+    }
+
+    &--very-completed:after{
```

Comment:
could this and  "&--completed:after" become 1 - because they're the same - or is that inconsistent

Also, most of these are mostly the same except for background color - could most of the styling going into and outer div or something that encompasses them?